pkg/providers/tinkerbell/stack/stack.go

```diff
@@ -210,3 +208,11 @@ func (s *Installer) uninstallBootsFromDocker(ctx context.Context) error {
 
 	return nil
 }
+
+func getURIDir(uri string) (string, error) {
+	index := strings.LastIndex(uri, "/")
+	if index == -1 {
+		return "", fmt.Errorf("uri is invalid: %s", uri)
```

Comment:
which uri is invalid? little more descriptive error would be good

Reply:
It does print out the URI. This is just a generic helper. I added more descriptive message from where this function gets called. Let me know what else I can add here

Reply:
oh missed that, ok